cli/cmd/customer_update.go

```diff
@@ -62,28 +67,49 @@ func (r *runners) updateCustomer(cmd *cobra.Command, _ []string) (err error) {
 		r.args.customerUpdateType = "prod"
 	}
 
-	channelID := ""
-	if r.args.customerUpdateChannel != "" {
+	channels := []kotsclient.CustomerChannel{}
+
+	foundDefaultChannel := false
+	for _, requestedChannel := range r.args.customerUpdateChannel {
 		getOrCreateChannelOptions := client.GetOrCreateChannelOptions{
 			AppID:          r.appID,
 			AppType:        r.appType,
-			NameOrID:       r.args.customerUpdateChannel,
+			NameOrID:       requestedChannel,
 			Description:    "",
-			CreateIfAbsent: r.args.customerUpdateEnsureChannel,
+			CreateIfAbsent: r.args.customerCreateEnsureChannel,
 		}
 
 		channel, err := r.api.GetOrCreateChannelByName(getOrCreateChannelOptions)
 		if err != nil {
 			return errors.Wrap(err, "get channel")
 		}
 
-		channelID = channel.ID
+		customerChannel := kotsclient.CustomerChannel{
+			ID: channel.ID,
+		}
+
+		if r.args.customerCreateDefaultChannel == requestedChannel {
+			customerChannel.IsDefault = true
+		}
+
+		channels = append(channels, customerChannel)
+	}
+
+	if len(channels) == 0 {
+		return errors.New("no channels found")
+	}
+
+	if !foundDefaultChannel {
+		// if the user didn't specify a default channel, the first channel will be the default
```

Comment:
I think this should either print a warning, or require default channel when multiple channels are used. It will not be obvious to the vendor.

Reply:
Having the default channel first is consistent with our other APIs.  I think a `log` here wold help clarify things.